gui/widgets/label.py

```diff
@@ -39,7 +44,17 @@ def show(self):
         super().show()  # Draw or erase border
         wri = self.writer
         dev = self.device
-        Writer.set_textpos(dev, self.row, self.col)
+        if self.align == ALIGN_LEFT:
+          Writer.set_textpos(dev, self.row, self.col)
```

Comment:
Please use 4 spaces for indentation rather than two: this is the standard used throughout the library.

Reply:
Sorry for that, I've corrected it.